### ISSUE TITLE
Login broken when using a TFA-enabled account

### ISSUE THREAD

**Issue by DylanGore** [Inrego/AbooIssueTracking#17]
Two-factor authentication is now supported by Plex accounts. When using an TFA-enabled account, the error "Unsuccessful response from 3rd party API." when attempting to log in.

**Comment by Inrego** [comment]
Yeah I did see the news, and I wouldn't expect it to work with Aboo.
Authenticating with Plex is already a big, confusing, undocumented mess, and I expect many other apps with Plex auth also don't support 2FA accounts yet.
I'll keep this issue open until I find a fix.
